tests/Console/ModelsCommand/LaravelCustomCasts/ReturnTypeTest.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types=1);
+
+namespace Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\LaravelCustomCasts;
+
+use Barryvdh\LaravelIdeHelper\Console\ModelsCommand;
+use Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\AbstractModelsCommand;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Foundation\Application;
+use Mockery;
+
+class ReturnTypeTest extends AbstractModelsCommand
+{
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (version_compare(Application::VERSION, '7.0', '<')) {
+            $this->markTestSkipped('This test requires Laravel 7.0 or higher');
+            return;
```

Comment:
Not needed, `markTestSkipped` works through exception bubbling

Reply:
found it only in `AbstractModelsCommand` but it checks for 6 version
```php
if (version_compare(Application::VERSION, '6.0', '<')) {
      $this->markTestSkipped('This test requires Laravel 6.0 or higher');
      return;
}
```

are you sure i can safely remove it? where is this exception debugging you are talking about?

Reply:
`markTestSkipped` throws a special exception recognized by PHPUnit, that's why a return (read: code afterwards) doesn't execute => https://github.com/sebastianbergmann/phpunit/blob/cb6b2d8cb5865b9576adcd9c595a8b4d005361b3/src/Framework/Assert.php#L2645-L2655

Reply:
yeah but to be honest, i cant see why i can drop this code
parent class check for 6 version, but my test should run only in 7

Reply:
It can/should be removed in the parent as well.

Reply:
sorry, but i don't follow why
travis runs test on multiple laravel versions, how else this test would be skipped without this code?

Reply:
or are you talking only about line 20? we can remove return statement, agreed

Reply:
Yes, exactly; apologies for being not more specific upfront!